src/testsuite/vec_dummy_main.c

```diff
@@ -20,7 +20,7 @@
       Created on: Nov 15, 2017
  */
 
-#ifndef __clang__
+#ifndef PVECLIB_DISABLE_F128MATH
 /* Disable for __clang__ because of bug involving <floatn.h>
    incombination with -mcpu=power9 -mfloat128 */
```

Comment:
delete this comment?

Reply:
I think I will keep this. Otherwise not obvious.